packages/drivers/routerlicious-driver/src/test/restWrapper.spec.ts

```diff
@@ -22,8 +23,16 @@ describe("RouterliciousDriverRestWrapper", () => {
     const token2 = "9876-auth-token-zyxw";
     const token3 = "abc-auth-token-123";
     let tokenQueue: string[] = [];
-    // Pop a token off tokenQueue to generate an auth header
-    const getAuthHeader = async () => `Bearer ${tokenQueue.shift() ?? ""}`;
+
+    const getToken: (refresh?: boolean) => Promise<ITokenResponse | undefined>
+        = async (refresh?: boolean) => new Promise((resolve) => {
+            // Pop a token off tokenQueue
+                const newToken: ITokenResponse = {
+                jwt: tokenQueue.shift() ?? "testtoken"
+            };
+            resolve(newToken);
+        });
```

Comment:
nit: you don't have to wrap this in a Promise with resolve. Simply declaring the function as `async` should be enough.